template/type.tpl

```diff
@@ -22,6 +22,26 @@
     {{ safe (renderComments .CommentLines) }}
 </p>
 
+{{ with (constantsOfType .) }}
+<table>
+    <thead>
+        <tr>
+            <th>Value</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+      {{- range . -}}
+      <tr>
+        {{- /* renderComments implicitly creates a <p> element, so we do the same here to make the value line up nicely.  */ -}}
```

Comment:
I don't get what this comment says, can you clarify the problem again please?

Reply:
`renderComments` wraps the comments in a `<p>` element via the markdown rendered. In a table row, if one cell is wrapped in a `<p>` but the other isn't they misalign and it looks bad.

Reply:
Hopefully this is clearer now.